lib/web_ui/test/canvaskit/initialization_test.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.12
+import 'dart:html' as html;
+import 'dart:typed_data';
+
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+
+import 'common.dart';
+
+void main() {
+  internalBootstrapBrowserTest(() => testMain);
+}
+
+void testMain() {
+  group('CanvasKit', () {
+    setUpCanvasKitTest();
+
+    test('populates flt-renderer and flt-build-mode', () {
+      final DomRenderer renderer = DomRenderer();
+      expect(html.document.body!.attributes['flt-renderer'], 'canvaskit (requested explicitly)');
+      expect(html.document.body!.attributes['flt-build-mode'], 'debug');
+    });
+    // TODO: https://github.com/flutter/flutter/issues/60040
```

Comment:
referring to closed issue ?

Reply:
That issue was closed by mistake (it has been reopened now; thanks for catching it!). We still need to skip iOS Safari until we switch to the new Simulator version.